android/app/src/main/java/com/extremeprogramming/financetracker/ui/reports/ReportsViewModel.kt

```diff
@@ -1,13 +1,26 @@
 package com.extremeprogramming.financetracker.ui.reports
 
-import androidx.lifecycle.LiveData
-import androidx.lifecycle.MutableLiveData
-import androidx.lifecycle.ViewModel
+import android.app.Application
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.*
+import com.extremeprogramming.financetracker.db.AppRepository
+import com.extremeprogramming.financetracker.db.entities.CategoryWithRecords
+import com.extremeprogramming.financetracker.db.entities.RecordWithCategory
+import java.util.*
+
+class ReportsViewModel(application : Application) : AndroidViewModel(application) {
+
+    private val currentMonth : Int = java.time.LocalDateTime.now().monthValue
+
+    private val repository : AppRepository = AppRepository.getRepository(application)
```

Comment:
Toto by mohol dostavat ako parameter ten viewModel. Lebo ak budeme chciet napriklad podsunut TestAppRepository tak musime zmenit to getRepository alebo to samotne priradenie.

Da sa to cez libku (Dagger 2) ale mi to mozme aj trosku manualnejsie (s daggerom nemam vobec skusenosti)
<img width="801" alt="image" src="https://user-images.githubusercontent.com/18246213/81497698-58d5bc80-92c0-11ea-9ff5-e7d4a17a9814.png">

<img width="519" alt="image" src="https://user-images.githubusercontent.com/18246213/81497704-5ffcca80-92c0-11ea-9adb-d4a3c8951696.png">

<img width="725" alt="image" src="https://user-images.githubusercontent.com/18246213/81497738-a3efcf80-92c0-11ea-9d72-1afc58a8f2bd.png">